Bottom: 3d238ca226c22717aff00f6f2cd75adbf04da59e
Top:    ed3e462f432bb461e205603d22e23255d7fc860e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-12-27 18:12:22 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
